### ISSUE TITLE
Active owner badly needed

### ISSUE THREAD

**Issue by brodybits** [Semantic-Org/Semantic-UI#6418]
From #4886, #6109, and other discussions I think it is clear that this project has no active owner. I am BEGGING @levithomason, @jlukic, or any other project owner to PLEASE add at least two more trusted owners or managers with the power to add more committers and maintainers so badly needed.

**Comment by levithomason** [comment]
@jlukic is the benevolent dictator here.  I'll let him respond, although, I think he's made his position pretty clear on it.

I've suggested a few times before that an active group of folks could always come together, fork, and start making releases.  My time is overbooked working on the React port and the v2 effort, in which we'll take on our own styling.

**Comment by hammy2899** [comment]
@danatemple This has already been discussed here https://github.com/Semantic-Org/Semantic-UI/issues/6109

**Comment by danatemple** [comment]
@hammy2899 yes I know. The level of donations was described there as being very low. However, this might have something to do with the fact that there are very few nudges to donate. There is a button right at the bottom of the entry webpage - but after you get to the docs, nothing (and you never go back to the main entry page, right?). There is no mention of donating on the GitHub readme, nor on the one for SUIR.

Seeing as it was possible to get $1M from kickstarter for some icons (https://www.kickstarter.com/projects/232193852/font-awesome-5/), I'm just suggesting that there is maybe some untapped potential.

**Comment by hammy2899** [comment]
I agree it could be pushed more but this will still be up to Jack.

We should also think about funding platforms like https://opencollective.com

**Comment by jlukic** [comment]
Just to be clear, my absence from frequent updates is specifically to address the need of finding a source of permanent funding.

Solving UI for the web through open source and SUI is the core goal of my professional life. It is an unfortunate necessity that I must leave things on pause for the time being, but I think this approach has a much greater chance to provide permanent funding for the project and its underlying goals, beyond what might be achieved by asking for donations or crowdfunding.

**Comment by brodybits** [comment]
@jlukic, understood. The one thing we need from you and @levithomason, DESPERATELY, is hopefully less than 20 minutes of your time to give 2 actively trusted people the power to add and remove privileged maintainers within this project and ideally within this organization. They should also have the power to add more administrators like themselves if needed. (Most privileged maintainers should not have such extra administrator power.)

I think this one action would be a MAJOR TECHNICAL LIFESAVER:
- prevent the otherwise inevitable project fork that is under discussion here, #6413, and #6109, among other places
- keep this project at the existing quality level
- avoid the dead project syndrome

By "project fork" I mean a significant portion of the team split off, like what happened with node.js in the past. I think it would be much better to keep this project as one united team if possible.

@jlukic I would like to commend you for the major efforts it must have taken on your part to get this project this far. I think we all understand 100% that you must be completely swamped with your professional work, business work for the project, and personal obligations. I remain extremely hopeful that you can help the rest of us with the one action proposed here.

**Comment by douglasg14b** [comment]
@jlukic 

The project needs lifesaving measures as it's community is dissatisfied enough to be discussing forks. I want to see this project live, and if forking it is the only way to get maintainers that then I'm behind it... You have good intentions for your project, and want to see it succeed. However, it's being smothered by inactivity and the inability of other members to step in. 

The `dead project syndrome` is already rearing it's head when pull requests to fix bugs are being auto-closed due to inactivity...

**Comment by ivantcholakov** [comment]
@jlukic At least stop the robot meanwhile.

**Comment by prudho** [comment]
@jlukic I'm sorry but I have to be this guy... The fact is that you left this repository unanswered for two plain month. Issues where opened and closed by the bot without any reponse, and nobody but you can do some critical things (merging pull requests, publishing fixes...).

I can understand that you don't have time to do SUI technical stuff, and I can understand your desire to make SUI a web standard and a long goal life. But you MUST understant that without repo's activity people will not be interested in SUI anymore, and you'll try to find funds for a dead project...

**Comment by donaggio** [comment]
I second @prudho opinion: please @jlukic let people who already volunteered to have a more active role in this project, at least merging PR for small bug fixes, updating FA icons as mentioned on another issue and stuff like that. That will be sufficient until you manage to sort out your long term goals.

**Comment by danatemple** [comment]
@jlukic Jack, I trust to your judgement in this. This is the same judgement that produced the design decisions in Semantic-UI after all :)

Give the guy some appreciation everyone!

I have Semantic-UI-React about to be deployed and it has been excellent to work with. Just stuck with the 2.2 CSS as described here https://react.semantic-ui.com/usage

**Comment by jlukic** [comment]
I've disabled the bot. It won't be trolling the issues board anymore.

I think the final vision of UI requires a cohesive system for theming, layouts, & ui, with simple, unified integrations for all js frameworks, and a well-defined toolkit to extend & distribute additional components, share themes and track versions between teams.  

I think this is something I can produce and distribute open-source in the long-term, but will be a project much larger than SUI 2.x, and will require me pressing on for some time longer.

I hope people keep that vision in mind, when considering the near-term goals like providing prompt feedback directly on GitHub discussions, and quick resolution for user-contributed PRs, maintenance releases etc. It hurts me that I'm not able to keep up the same pace as I had when 2.0 was launched. I hope you guys keep that in mind, and be generous in your views

**Comment by donaggio** [comment]
@jlukic thank you for clarifying your long-term vision for SUI, and I really really like it! But still, it doesn't seems totally incompatible with the much more short-term plan of letting other talented people maintain SUI 2.x while you work towards what it seems to be SUI 3.x / SUI-NG, don't you think?

Anyway, thanks for all your hard work and dedication to this project!

**Comment by arjarn** [comment]
@jlukic  : agree with prudho, donaggio (and many others) : tasks as bug fix, minors improvments, answering question about using, writing documentation are taking lot of times. It seem you've big community who love SUI and agree your mind. People are ready to help u for improving this great framework. 
Giving all this things to other and keeping your time for deeping development may be win-win for all :
 - for you, cause you can use 100% time for your (great) goal
 - for community cause talented people are ready to help
 - for users cause they'll find answers and see project is alive and active (what's more frustrating / dissuasive for dev to see dead repo ? almost if they are professional dev or Decision-makers in enterprise).

I agree your mind but really sad about results...  :-|

**Comment by peterrobertz** [comment]
I love you Jack, for making SUI. I just hope you dont get crazy <3

**Comment by Atulin** [comment]
Can we officially pronounce SemanticUI dead yet, or...
![image](https://user-images.githubusercontent.com/11233299/44952087-2ed6c080-ae76-11e8-92c8-9e312ac971d9.png)

**Comment by dreaming-augustin** [comment]
In reply to the previous comment, I'd like to say the same thing I said here: 
https://github.com/Semantic-Org/Semantic-UI/issues/6413#issuecomment-418018371

**Comment by jlukic** [comment]
@Atulin those are commits in master. Look at the `next` branch and twitter.

**Comment by davegson** [comment]
@jlukic

> Solving UI for the web through open source and SUI is the core goal of my professional life—a project I expect to continue for many more decades to come.

I do applaud that! 🎉 I like ambitious goals. At the same time I must mention one thing: **you will not succeed on your own**.

In that regard I find it highly confusing - and disrespectful imho - that you chose to ignore all suggestions to delegate power to trusted contributors who would like to help you to maintain SUI 2.x. Which would _reduce_ your already high work load.

### Solving UI for the web

This task is too great to be solved by a single person. The key here is not to solely depend on your brainpower - instead, multiply by giving others authority. I know you know this. You already talked with others about splitting up the workload, but your actions currently tell another story.

If "Solving UI for the web" **truly** is your goal, then I'd advise you to rethink your stance on delegation. You already showed you have what it takes to do this! You built this awesome ship which *could* solve UI for the web. But it's gotten too big for you to micro manage every damn aspect of it. Let others help you with its navigation!

PS: You will still be the captain ;)
